src/OrbitCaptureClient/include/OrbitCaptureClient/ApiEventProcessor.h

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_CAPTURE_CLIENT_API_EVENT_PROCESSOR_H_
+#define ORBIT_CAPTURE_CLIENT_API_EVENT_PROCESSOR_H_
+
+#include <absl/container/flat_hash_map.h>
+
+#include "Api/EncodedEvent.h"
+#include "OrbitCaptureClient/CaptureListener.h"
+#include "capture.pb.h"
+
+class ApiEventProcessor {
```

Comment:
This file needs the namespace.

Reply:
which namespace?

Reply:
I would have said `orbit_capture_client`, but it seems that the whole module lacks the namespace so I guess it's fine as it is.